src/state/GlobalState.tsx

```diff
@@ -42,6 +51,7 @@ interface Props {
 interface StateProps {
   category: string
   data: GameInfo[]
```

Comment:
Not necessary now but in the future would be good to rename this one to epicLibrary.